documentation/release_notes.rst

```diff
@@ -8,6 +8,26 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2022.4.0
+===============
+
+New Features
+------------
+- Added highly configurable ``radix_sort`` and ``radix_sort_by_key`` algorithms
+  residing in ``oneapi::dpl::experimental::kt::esimd`` namespace, which work with Intel® Data Center GPU Max Series.
+  They can be parameterized by a number of elements processed by a work-item,
+  number of work-items in a work-group to achieve the best performance depending on
+  input characteristics such as a number of elements and their type.
+- Added new ``transform_if`` algorithm for conditionally applying a transform based on
+  a predicate to the input sequence(s). Overloads exist for a single input sequence with
+  a unary transform and unary predicate and for a pair of input sequences with
+  a binary transform and binary predicate.
```

Comment:
```suggestion
- Added new ``transform_if`` algorithm for applying a transform function conditionally
  based on a predicate, with overloads provided for one and two input sequences
  that use correspondingly unary and binary operations and predicates.
```

Reply:
Thanks @akukanov , I think this is a more clear and concise way of putting it.